ml/train/losses/losses.go

```diff
@@ -21,8 +21,12 @@
 package losses
 
 import (
+	"fmt"
+
 	. "github.com/gomlx/exceptions"
+	"github.com/gomlx/gomlx/graph"
```

Comment:
So, in files that involve numeric algorithms in GoMLX, like defining losses, the convention has been to import `graph` package as ".", so there is no need to add `graph.` everywhere.